pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -220,7 +220,7 @@ buildStdenv.mkDerivation ({
     "profilingPhase"
   ];
 
-  patches = lib.optionals (lib.versionOlder version "103") [
+  patches = lib.optionals (lib.versionOlder version "102.6.0") [
```

Comment:
I removed this patch too early previously, that's when thunderbird failed to build.